### ISSUE TITLE
Ability to ignore/exclude a specific plugin in redundantPlugins check

### ISSUE THREAD

**Issue by eugene-krivobokov** [autonomousapps/dependency-analysis-android-gradle-plugin#427]
**Is your feature request related to a problem? Please describe.**

I want to enable `redundantPlugins` check for all modules:

```kotlin
dependencyAnalysis {
    issues {
        all {
            onRedundantPlugins {
                severity("fail")
            }
        }
    }
}
```

For some modules, I got false-positive errors: #392, #411
I'm ok with it. False-positive is inevitable for any linter tool.

**Describe the solution you'd like**

I'd like to ignore a specific plugin in a module.
Probable API:

```kotlin
project(":problematic-module") {
   issues {
            onRedundantPlugins {
                exclude("java-library") // `java-library` is added implicitly by `java-gradle-plugin`
            }
    }
}
```

**Describe alternatives you've considered**

I have to ignore the check:

```kotlin
project(":problematic-module") {
   issues {
            onRedundantPlugins {
                severity("ignore")
            }
    }
}
```

The task is not failing but still prints the error:

```
Plugin advice:
- java-library: this project has both java-library and org.jetbrains.kotlin.jvm applied, which is redundant. You can remove java-library
```

**Additional context**

Plugin version: 0.74.0
You can reproduce it in this project: https://github.com/avito-tech/avito-android/tree/dependency-analysis-reproducer-427 

```bash
./gradlew -p subprojects :gradle:build-properties:projectHealth
```

**Comment by autonomousapps** [comment]
Thanks for the report. The printed error looks like a bug, but the rest is a FR. I think it makes sense.

**Comment by eugene-krivobokov** [comment]
Should I create a separate issue for printed error? It looks like a `good first issue`.

**Comment by autonomousapps** [comment]
Sure, that makes sense.

**Comment by eugene-krivobokov** [comment]
#431

**Comment by KatieBarnett** [comment]
I'm experiencing the same issue with kotlin-kapt.

- side note on this, for some of my modules kotlin-kapt is reported as redundant but is required when building, I'll raise an issue if I can isolate in a reproducible project.

**Comment by autonomousapps** [comment]
@KatieBarnett you might be facing https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/issues/411

I would certainly appreciate a reproducer as that would be key in helping to resolve this issue.
